nixos/tests/kbd.nix

```diff
@@ -0,0 +1,20 @@
+import ./make-test-python.nix ({ pkgs, ... }: {
+  name = "kbd";
+
+  machine = { pkgs, ... }: { };
+
+  testScript = ''
+    machine.wait_for_unit("default.target")
+
+    assert "${pkgs.kbd.version}" in machine.succeed("loadkeys -V")
+    # test if keymaps are installed
+    machine.succeed("loadkeys us")
+    machine.succeed("loadkeys fr")
+    machine.succeed("loadkeys de")
+    machine.succeed("loadkeys neo")
+    machine.succeed("loadkeys bone")
+    machine.succeed("loadkeys es-olpc")
+    machine.succeed("loadkeys dvorak")
+    machine.succeed("loadkeys colemak")
```

Comment:
As mentioned in my comment to @ofborg, we already have a [`keymap` test](https://github.com/NixOS/nixpkgs/blob/4b21d06df01d181ad91f5ba224b9cd3819f4949a/nixos/tests/keymap.nix) which not only tests `loadkeys` but tests whether the mapping is correctly applied. At the moment only `azerty`, `colemak`, `dvorak`, `dvp`, `neo` and  `qwertz` is tested, so if you're using another layout, feel free to add a subtest there.

Reply:
i haven't found that. thanks!